doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -96,15 +96,19 @@ features:
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
-    Bluetooth LE Co-existence:
+    Bluetooth LE coexistence:
       rule: WIFI_NRF700X && NRF700X_SR_COEX
       boards_and_shields:
         - SHIELD_NRF7002EK
         - SHIELD_NRF7002EK_NRF7001
         - SHIELD_NRF7002EB
         - BOARD_NRF7002DK_NRF5340_CPUAPP
         - BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001
-    SoftAP Mode:
+    Thread coexistence:
+      rule: WIFI_NRF700X && NRF700X_THREAD_COEX
```

Comment:
rather than a dummy Kconfig we can use `WIFI_NRF700X && OPENTHREAD` this is the combination for OTBR where thread co-ex comes in to picture.

Reply:
Also the SoftAP change should be in a separate commit, it's not matching commit log

Reply:
I have modified the same as provided in the comment and moved SoftAP and bluetooth co-existence naming to a new commit